src/bindings/js/node/tests/unit/utils.js

```diff
@@ -3,15 +3,56 @@
 // SPDX-License-Identifier: Apache-2.0
 
 const path = require('path');
+const { downloadFile, checkIfDirectoryExists } = require('../../scripts/download_runtime');
 
-module.exports = { getModelPath };
+const modelDir = 'tests/unit/test_models/';
+const testModels = {
+  testModelFP32: {
+    xml: 'test_model_fp32.xml',
+    bin: 'test_model_fp32.bin',
+    xmlURL:
+    'https://raw.githubusercontent.com/openvinotoolkit/testdata/master/models/test_model/test_model_fp32.xml',
+    binURL:
+    'https://media.githubusercontent.com/media/openvinotoolkit/testdata/master/models/test_model/test_model_fp32.bin',
+
+  },
+};
+
+module.exports = {
+  getModelPath,
+  downloadTestModel,
+  checkTestModel,
+  testModels,
+};
 
 function getModelPath(isFP16=false) {
-  const basePath = 'tests/unit/test_models/';
   const modelName = `test_model_fp${isFP16 ? 16 : 32}`;
 
   return {
-    xml: path.join(basePath, `${modelName}.xml`),
-    bin: path.join(basePath, `${modelName}.bin`),
+    xml: path.join(modelDir, `${modelName}.xml`),
+    bin: path.join(modelDir, `${modelName}.bin`),
   };
 }
+
+async function downloadTestModel(model) {
+  const baseArtifactsDir = './tests/unit/test_models';
+  const model_path = path.join(baseArtifactsDir, model.xml);
+  const modelExists = await checkIfDirectoryExists(model_path);
+  if ( !modelExists ) {
+    const { env } = process;
+    const proxyUrl = env.http_proxy || env.HTTP_PROXY || env.npm_config_proxy;
+
+    await downloadFile(model.xmlURL, baseArtifactsDir, model.xml, proxyUrl);
+    await downloadFile(model.binURL, baseArtifactsDir, model.bin, proxyUrl);
+  }
+}
+
+async function checkTestModel(model) {
```

Comment:
What does this function check? How about names: `isModelPresent` or `isModelAvailable`?